### DISCUSSION TITLE
POST_INIT, POST_SERVICE working but host Section doesn't

### DISCUSSION THREAD

**Discussion by Soylor** [ltsp/ltsp#799] [General] [answered]
Cannot get working one `ltsp.conf `section
This Works
` POST_SERVICE_AD="/etc/init.d/stc0 172.19.46.4 dominio permisos_ro epoptes"
`
This Works too
```
 POST_INIT_TUNE="section_tunning"
[tunning]
 sed -i 's/"1"/"0"/' /etc/apt/apt.conf.d/20auto-upgrades
 sed -i 's/Prompt=.*/Prompt=never/' /etc/update-manager/release-upgrades
 /etc/init.d/stc0 172.19.46.4 home_alumno
```
This does not work (cannot ejecute etc/init.d/stc0 )
```
[host01]
 /etc/init.d/stc0 172.19.46.4 epoptes

```
Why is [host] commands not working?

Note: `/etc/init.d/stc0 `is a bash script

**Comment by alkisg** [comment, marked as answer]
Because `[host]` sections are not designed to accept commands.
They are designed to accept directives, such as POST_INIT_x.
Then ltsp sources these sections MANY times, e.g. in `ltsp initrd-bottom`, in `ltsp init`, in `ltsp service` etc.
You wouldn't want to run these commands ALL these times. That's why you're supposed to use directives and not commands.
Then ltsp can see if you're using POST_INITRD_BOTTOM_x, or POST_INIT_x, or POST_SERVICE_x, and then it will execute the commands only on that applet, not on all of them.

**Reply by Soylor** [reply]
I understand. Thank you very much.

Got working changig:

```
[host01]
 /etc/init.d/stc0 172.19.46.4 epoptes

```
To:

```
[host01]
 POST_SERVICE_h1="/etc/init.d/stc0 172.19.46.4 epoptes"
```
